From: Michael Biebl <biebl@debian.org>
Date: Mon, 26 Nov 2018 23:29:08 +0100
Subject: Move D-Bus policy file to /usr/share/dbus-1/system.d/

This is supported since dbus 1.9.18
The old location in /etc/dbus-1/system.d/ has been deprecated.

https://lists.freedesktop.org/archives/dbus/2015-July/016746.html

https://gitlab.freedesktop.org/upower/upower/merge_requests/23
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index d311c7a..3400139 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -163,7 +163,7 @@ dbusservice_DATA     = $(dbusservice_in_files:.service.in=.service)
 $(dbusservice_DATA): $(dbusservice_in_files) Makefile
 	@sed -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
 
-dbusconfdir = $(sysconfdir)/dbus-1/system.d
+dbusconfdir = $(datadir)/dbus-1/system.d
 dbusconf_in_files = org.freedesktop.UPower.conf.in
 dbusconf_DATA = $(dbusconf_in_files:.conf.in=.conf)
 
